### PR TITLE
Remove files_locking from shipped.json

### LINKED ISSUE #22083: Remove files_locking from shipped.json
Hi,

it seems files_locking is still listed as shipped in:

https://github.com/owncloud/core/blob/master/core/shipped.json#L15

but its not even included in recent oC versions (8.2.2, 9.0.0).

Furthermore are there any regressions if people are using the new file locking included in core and the old files_locking app?

### PR DESCRIPTION
**Pull request by ghost**
Fixes #22083 

~~files_antivirus is also not shipped anymore.~~

### CODE REVIEW

**Comment by LukasReschke** [comment]
> files_antivirus is also not shipped anymore.

I suppose it's in there because of EE? 

cc @MTRichards @karlitschek

**Comment by MTRichards** [comment]
I thought EE shipped with files_antivirus?

**Comment by RealRancor** [comment]
You're the guys having access to the EE :-)

If the files_antivirus is shipped with EE then the shipped.json probably should differ between the CE and EE as the files_antivirus is not shipped anymore in the CE.

**Comment by nickvergessen** [comment]
Well we don't distinguish, there are also a lot of other apps in that list which are not in CE. The list is only used to know which apps should not get disabled while updating, so I think it's okay to have them in one list.

File: core/shipped.json [hunk 1]
```diff
@@ -8,11 +8,9 @@
     "external",
     "federation",
     "files",
-    "files_antivirus",
```
Comment (nickvergessen) [inline core/shipped.json]: Still shipped in EE, so this needs to stay

**Comment by RealRancor** [comment]
Yes, but the shipped.json is also used in the CE and won't disable the files_antivirus apps on upgrades which might break such an upgrade. As this is a different issue i've raised https://github.com/owncloud/core/issues/22092

File: core/shipped.json (hunk 1 above)
Comment (RealRancor) [inline core/shipped.json]: Added again

**Comment by nickvergessen** [comment]
:+1:

**Comment by rullzer** [comment]
:+1:

**Comment by karlitschek** [comment]
tricky. I agree with @nickvergessen for now. But in the next release we should consider to also apps.owncloud.com also enabled for EE so simplify this.  :+1:

**Pull request merged**